R/visualization/plotBestMethodsRNA.R

```diff
@@ -1,25 +1,58 @@
+#' Plotting scib-metrics result of the integration across multiple RNA tasks in an
+#'  overview plot that shows the best-performing combination of pre-processing choices for 
+#'  each integration method.
+#'
+#' \code{plotBestMethodsRNA} saves in `outdir` the overview table in three formats (.pdf/.tiff/.png)
```

Comment:
Usually you would put this kind of output stuff in the `@return` section but this is still clear so not essential.